modules/mandrel/20.2.0.0.Beta1-java11/module.yaml

```diff
@@ -0,0 +1,29 @@
+schema_version: 1
+name: mandrel
+version: &version "20.2.0.0.Beta1-java11"
+
+labels:
+  - name: mandrel-archive-filename
+    value: &filename mandrel-java11-linux-amd64-20.2.0.0.Beta1.tar.gz
+  - name: mandrel-archive-url
+    value: &url https://github.com/graalvm/mandrel/releases/download/mandrel-20.2.0.0.Beta1/mandrel-java11-linux-amd64-20.2.0.0.Beta1.tar.gz
+
+envs:
+  - name: "JAVA_HOME"
+    value: "/opt/mandrel"
+  - name: "GRAALVM_HOME"
+    value: "/opt/mandrel"
+  - name: "FILENAME"
+    value: *filename
+
+artifacts:
+- name: *filename
+  url: *url
+  md5: a341927dfae93a217dda5a8915bb129e
```

Comment:
Using the md5 hash of 20.1.0.2.Final makes the build succeed, but using the wrong archive.

Reply:
OK, thanks. I'll update the PR once the releases are available.

Reply:
Fixed md5 sums for both. Releases are now available. Should be ready for integration once build passes.

Reply:
@cescoffier we probably need to add an extra step to run the image and print the `native-image --version` to avoid issues like this in the future :)